index.html

```diff
@@ -0,0 +1,213 @@
+    rel="stylesheet">
+  <!-------------------------swiper js----------------------------->
+
+</head>
+
+<body>
+  <header>
+    <nav>
+      <div class="logo">
+        <a href="#home">
+          <img src="img/logo.png" alt="" class="logo">
+        </a>
+      </div>
+      <div class="humbergur"><i class="fa-solid fa-bars"></i></div>
+      <ul class="ul-items">
+        <li class="active"><a href="index.html">Home</a></li>
+        <li><a href="about.html">About us</a></li>
+        <li><a href="#activity">Activity</a></li>
+        <li><a href="#team">Staff</a></li>
+        <li><a href="#partner">Partner</a></li>
+        <li class="enroll"><a href="#">Enrollment</a></li>
+      </ul>
+    </nav>
+  </header>
+  <div class="wrapper">
+    <div class="overlay">
+      <div class="popup">
+        <div class="title">
+          <h3>Enrollment Form</h3>
+          <p class="closeBtn">&times;</p>
+        </div>
+        <hr class="line">
+        <form action="https://formspree.io/f/mpzbknzr" method="post">
+          <div class="double">
+            <div class="mb-3">
+              <label for="fname" class="form-label">Firstname</label>
+              <input type="text"  placeholder="Enter Firstname" name = "fname" class="form-control" id="fname" >
+            </div>
+            <div class="mb-3">
+              <label for="lname" class="form-label">Lastname</label>
+              <input type="text"  placeholder="Enter Lastname" name = "lname" class="form-control" id="lname" >
+            </div>
+          </div>
+          <div class="double">
+            <div class="mb-3">
+              <label for="email" class="form-label">Email address</label>
+              <input type="email" placeholder=" Enter Email address" name = "email"  class="form-control" id="email" >
+            </div>
+            <div class="mb-3">
+              <label for="phone" class="form-label">Contact</label>
+              <input type="text"  placeholder="Enter Contact" name = "phone" class="form-control" id="phone" >
+            </div>
+          </div>
+          <div class="double">
+            <div class="mb-3">
+              <label for="pass" class="form-label">Password</label>
+              <input type="password" placeholder="Enter Password" name = "pass"  class="form-control" id="pass">
+            </div>
+            <div class="mb-3">
+              <label for="cpass" class="form-label">Confirm Password</label>
+              <input type="password" placeholder="Confirm Password" name = "cpass"  class="form-control" id="cpass">
+            </div>
+          </div>
+          <button type="submit" class="btn btn-primary">Submit</button>
+        </form>
+      </div>
+    </div>
+    <section class="home-pg" id="about">
+      <img class=" circle r1" src="img/round.png" alt="">
+      <img class=" circle r4" src="img/r4.png" alt="">
```

Comment:
- [ ]  Kindly add alt tags to all images, please do this because if your image is not rendered due to network issues or a broken link, users can still see a text that says what the image about, this make your website more accessible